pyspark/export_to_parquet.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/python
```

Comment:
Perhaps we should put this in a `script/pyspark` folder in case we have other types of scripts we want to codify in this repo?

Reply:
sure, wfm